eng/Directory.Packages.props

```diff
@@ -229,6 +229,7 @@
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="ICSharpCode.Decompiler" Version="8.1.1.7464" />
     <PackageVersion Include="Microsoft.CodeAnalysis.Elfie" Version="1.0.0" />
+    <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
```

Comment:
So this needs to have a variable defined for it, and then consumed here otherwise we'll run into other source build issues.